src/ApiPort/ApiPort.csproj

```diff
@@ -13,6 +13,7 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <DefineConstants>DESKTOP_CONFIGURATION;RICH_CONSOLE</DefineConstants>
```

Comment:
The rest of the solution uses the convention FEATURE_* for define constants (such as FEATURE_DESKTOP_CONFIGURATION and FEATURE_RICH_CONSOLE). Can you switch these to follow that?